packages/jsii-pacmak/lib/targets/python.ts

```diff
@@ -1728,7 +1728,44 @@ class PythonModule implements PythonType {
     code.line('import publication');
     code.line('import typing_extensions');
     code.line();
-    code.line('from typeguard import check_type');
+
+    code.line('import typeguard');
+    code.line('from importlib.metadata import version as _metadata_package_version');
```

Comment:
This was [added in Python 3.8](https://docs.python.org/3/whatsnew/3.8.html#new-modules), which is the lowest version of python we support - so its safe to use. Aliasing to avoid possible conflicts with other generated code.